### ISSUE TITLE
Documentation Update

### ISSUE THREAD

**Issue by ArlindKadra** [openml/openml-python#1069]
https://github.com/openml/openml-python/blob/6e8a9db03fd1af9d3eb0623970101413b531cc69/openml/runs/run.py#L28-L63

This probably we should update, considering there are some fields that do not have a description and might be clearer with one.

**Comment by v-parmar** [comment]
Hey, What should you want to update?

**Comment by PGijsbers** [comment]
For each attribute we would want a description of what it represents, e.g.:
```diff
     trace: OpenMLRunTrace 
+     The trace containing information on internal model evaluations of this run.
     model: object 
+     The untrained model that was evaluated in this run.
```

**Comment by v-parmar** [comment]
Hey, I have done the given task. How can I submit it?

**Comment by PGijsbers** [comment]
We documented the process in our [contribution guidelines](https://github.com/openml/openml-python/blob/develop/CONTRIBUTING.md#general-git-workflow), in short:
 - push to your fork
 - check that it meets all the criteria to open a pull request (optionally add things if it doesn't)
 - set up a pull request and wait for review

**Comment by v-parmar** [comment]
Hey @PGijsbers, I have opened the PR on issue #1194 and added the description please check it.

**Comment by mfeurer** [comment]
This was solved by @v-parmar via #1194.
